orchestrator-salt/src/main/resources/salt/salt/gateway/config/cm/topology.xml.j2

```diff
@@ -83,6 +83,10 @@
             <param>
                 <name>impalaui.acl</name>
                 <value>*;{{ salt['pillar.get']('gateway:envAccessGroup') }};*</value>
+	    </param>
```

Comment:
Can we add a space here for better indentation ?

Reply:
Done. I had some issues with tabs and spaces, it would seem.